documentation/automating_windows_targets.md

```diff
@@ -168,269 +169,117 @@ documentation.](https://puppet.com/docs/pdk/1.x/pdk_overview.html)
-   ```
-
-1. Create a plan called `puppet_choco_tap::installer` by copying the following
-   code into the `installer.pp` file::
-
-   ```
-   plan puppet_choco_tap::installer(
-     TargetSpec $targets,
-     String $package,
-     Variant[Enum['absent', 'present'], String ] $ensure = 'present',
-   ){
-     apply_prep($targets)
-
-     apply($targets){
-       include chocolatey
-
-       package { $package :
-         ensure    => $ensure,
-         provider  => 'chocolatey',
-         }
-       }
-     }
-   }
-   ```
-   Take note of the following features of the plan:
-
-   - It has three parameters: the list of targets to install the package on, a
-     `package` string for the package name, and the `ensure` state of the package
-     which allows for version, absent or present.
-   - It has the `apply_prep` function call, which is used to install modules needed
-     by `apply` on targets as well as to gather facts about the targets.
-   - `include chocolatey` installs the Chocolatey package manager. The Chocolatey
-     provider is also deployed as a library with the Puppet agent in `apply_prep`.
-   - The [package
-     resource](https://puppet.com/docs/puppet/latest/types/package.html) ensures a
-     package's state using the Chocolatey provider.
-
-1. To verify that the `puppet_choco_tap::installer` plan is available, run the
-   following command inside the `bolt_choco_example` directory:
-
-   ```
-   bolt plan show
-   ```
-
-   The output should look like:
-
-   ```
-   facts
-   facts::info
-   puppet_choco_tap::installer
-   puppetdb_fact
-   ```
+    ```powershell
+    Invoke-BoltApply -Execute "include chocolatey"  -Targets <TARGET URI> -User <USER> -Password <PASSWORD> -Transport winrm
+    ```
 
-1. Run the plan with the `bolt plan run` command: 
+1. Use the built-in Package task to install Frogsay on your target:
+    
+    _\*nix shell command_
 
-   ```
-   bolt plan run puppet_choco_tap::installer package=frogsay --targets=windows
-   ```
+    ```shell
+    bolt task run package -t <TARGET URI> -u <USER> -p <PASSWORD> --transport winrm action=install name=frogsay
+    ```
 
-   The output looks like this:
+    _PowerShell cmdlet_
 
-   ```
-   Starting: plan puppet_choco_tap::installer
-   Starting: install puppet and gather facts on chocowin0.classroom.puppet.com, chocowin1.classroom.puppet.com
-   Finished: install puppet and gather facts with 0 failures in 22.11 sec
-   Starting: apply catalog on chocowin0.classroom.puppet.com, chocowin1.classroom.puppet.com
-   Finished: apply catalog with 0 failures in 18.77 sec
-   Starting: apply catalog on chocowin0.classroom.puppet.com, chocowin1.classroom.puppet.com
-   Finished: apply catalog with 0 failures in 33.74 sec
-   Finished: plan puppet_choco_tap::installer in 74.63 sec
-   ```
+    ```powershell
+    Invoke-BoltTask -Name package -Targets <TARGET URI> -User <USER> -Password <PASSWORD> -Transport winrm -Params @{action='install'} @{name='frogsay'}
```

Comment:
```suggestion
    Invoke-BoltTask -Name package -Targets <TARGET URI> -User <USER> -Password <PASSWORD> -Transport winrm action='install' name='frogsay'
```

Elsewhere in the docs we show passing params in the same manner as *nix shells, so I think it'd be good to maintain consistency here.

Reply:
@beechtom Does it still need the quotes? ie. `action='install'`

Reply:
Not if it's a single word, no.